tests/test_cli.py

```diff
@@ -635,3 +712,6 @@ def test_remove_settings_entry():
             ],
         )
         assert result.exit_code == 1
+if __name__ == "__main__":
+    log_file = tempfile.NamedTemporaryFile()
+    test_dry_activities(log_file)
```

Comment:
The `log_file` is already created for you in [`tests/conftest.py`](https://github.com/chaostoolkit/chaostoolkit/blob/master/tests/conftest.py) so you don't need to recreate it

```suggestion

```

Reply:
This whole block isn't needed also as `pytest` does not use `if __name__ == "__main__"` to run tests

Reply:
Yep, it was just for my tests. missed that before i pushed.